src/lib/vue-i18n.ts

```
@@ -5,18 +5,16 @@
 import i18nJA from "@/lang/ja";
 import i18nFR from "@/lang/fr";
 
-import * as constant from "@/config/constant";
+import { stripe_regions_jp } from "@/config/constant";
 
 import { ownPlateConfig } from "@/config/project";
 
 const region = ownPlateConfig.region || "US";
```

ESLint

[error] 'region' is assigned a value but never used. Allowed unused vars must match /^__/u.

(@typescript-eslint/no-unused-vars)